linera-views/src/test_utils/mod.rs

```diff
@@ -281,7 +281,12 @@ pub async fn run_reads<S: LocalKeyValueStore>(store: S, key_values: Vec<(Vec<u8>
         }
         assert_eq!(set_key_value1, set_key_value2);
     }
-    // Now checking the read_multi_values_bytes
+    // Now checking the `read_value_bytes`
+    for (key, value) in &key_values {
+        let value_read = store.read_value_bytes(key).await.unwrap();
+        assert_eq!(value_read, Some(value.clone()));
+    }
+    // Now checking the `read_multi_values_bytes`
```

Comment:
This seems unrelated to this PR, wouldn't it be better to have a separate PR for it?

Reply:
Ok, I put it in the multi read PR for ScyllaDb where it is a little better.